Makefile

```diff
@@ -13,17 +13,17 @@
 
 IMAGE_NAME := fluent/fluentd
 X86_IMAGES := \
-	v1.12/alpine:v1.12.4-1.0,v1.12-1,edge \
-	v1.12/debian:v1.12.4-debian-1.0,v1.12-debian-1,edge-debian
+	v1.12/alpine:v1.12.4-1.1,v1.12-1,edge \
+	v1.12/debian:v1.12.4-debian-1.1,v1.12-debian-1,edge-debian
 #	<Dockerfile>:<version>,<tag1>,<tag2>,...
 
 # Define images for running on ARM platforms
 ARM_IMAGES := \
-	v1.12/armhf/debian:v1.12.4-debian-armhf-1.0,v1.12-debian-armhf-2,edge-debian-armhf \
+	v1.12/armhf/debian:v1.12.4-debian-armhf-1.1,v1.12-debian-armhf-2,edge-debian-armhf \
```

Comment:
Note: `v1.12-debian-armhf-2` trailing `2` should be fixed in the next version (already done at #270)

Reply:
:sweat_smile: